### PR TITLE
resolves #43 Transform any string url in the bot message text into a clickable one

### LINKED ISSUE #43: [MessageBot] Transform any string url in the bot message text into a clickable one

### PR DESCRIPTION
**Pull request by broxmik**

### CODE REVIEW

File: src/components/MessageBot/MessageBot.tsx [hunk 1]
```diff
@@ -18,25 +18,41 @@ const Message: StyledComponent<{}, {}, TockTheme> = styled.div`
...
+                return '<a href="' + replaced + '" target="_blank">' + replaced + '</a>';
+            });
+        }
+        return message;
+    }
+
+    return (
+        <MessageContainer>
+            <Message>
+                <span dangerouslySetInnerHTML={{__html: getHtmlContent()}}/>
```
Comment (elebescond) [inline src/components/MessageBot/MessageBot.tsx:52]: If bot message contains "`<p>message</p>`" generating HTML becomes "`<span><p>message</p></span>`".

And element p not allowed as child of element span in this context.

Replacing span with a div element?

File: src/components/MessageBot/MessageBot.stories.tsx [hunk 2]
```diff
@@ -4,7 +4,9 @@ import MessageBot from './MessageBot';
 
 const message: string = 'A bot message';
 
-const html: string = '<b>A bot message</b>';
+const html: string = '<b>A html bot message</b> <br>' +
+    'with a html link to the <a href="https://doc.tock.ai">Tock Documentation Page</a><br>' +
+    'and a clickable string url: github.com/theopenconversationkit/tock-react-kit';
```
Comment (elebescond) [inline src/components/MessageBot/MessageBot.stories.tsx:9]:
```
const html: string = '<b>A html bot message</b> <br>' +
    'with a html link to the <a href="https://doc.tock.ai">Tock Documentation Page</a><br>' +
    'and a clickable string url: github.com/theopenconversationkit/tock-react-kit<br>vaenl1.com' +
    '<br><a href="tel:0612345678">tel</a>' +
    '<br><a href="mailto:raoul@gmail.com">mail</a><br />' +
    'Bonjour nom.prenom merci pour votre commande';
```

image

Comment (elebescond) [reply src/components/MessageBot/MessageBot.stories.tsx:9]: https://www.npmjs.com/package/linkifyjs
https://www.npmjs.com/package/get-urls

File: src/components/MessageBot/MessageBot.tsx (hunk 1 above)
Comment (broxmik) [reply src/components/MessageBot/MessageBot.tsx]: done! 👍

File: src/components/MessageBot/MessageBot.stories.tsx (hunk 2 above)
Comment (broxmik) [reply src/components/MessageBot/MessageBot.stories.tsx]: I replaced the "homemade" regex with linkifyjs.

**Pull request merged**